vnext/Microsoft.ReactNative/ReactInstanceSettings.h

```diff
@@ -174,6 +177,7 @@ struct ReactInstanceSettings : ReactInstanceSettingsT<ReactInstanceSettings> {
   hstring m_sourceBundleHost{};
   hstring m_debuggerRuntimeName{};
   uint16_t m_sourceBundlePort{0};
+  bool m_inlineSourceMap{true};
```

Comment:
Given that core never seems to specify inlineSourceMap we should probably make this default to false:

https://github.com/facebook/react-native/blob/062c1f7277ebaadda5e6aff1e839618caf0ea0ac/ReactAndroid/src/main/java/com/facebook/react/devsupport/DevServerHelper.java#L428

Reply:
Oh.. I see that's a change from before. - I guess we are already defaulting to true.  Probably shouldn't change it then.

Reply:
Correct, it was false specifically for Hermes before, once we get rid of that it'll be true everytime, so we made it explicitly default.